README.md

```diff
@@ -1,6 +1,8 @@
 GitLink
 ==========
 
+[![Build status](https://ci.appveyor.com/api/projects/status/g56b4xyl4q0ean8i/branch/develop?svg=true)](https://ci.appveyor.com/project/AArnott/gitlink/branch/develop)
```

Comment:
@GeertvanHorrik I have added the GitLink project on AppVeyor: https://ci.appveyor.com/project/GitTools/gitlink Not entirely sure whether this will start building the PR's yet, as there is not an appveyor.yml file in the default branch yet.  But we will soon find out. 😄